README.md

```diff
@@ -96,11 +103,21 @@ MIT Licensed. See [LICENSE](https://github.com/deliveryhero/tf-ssh-bastion/tree/
 
 | Name | Description |
 |------|-------------|
-| endpoint | The cluster endpoint |
-| instance_endpoints | A list of all cluster instance endpoints |
-| master_password | The master password |
-| master_username | The master username |
-| port | The port |
-| rds_cluster_id | The ID of the cluster |
-| reader_endpoint | The cluster reader endpoint |
-| security_group_id | The security group ID of the cluster |
+| this_rds_cluster_endpoint | The cluster endpoint |
+| this_rds_cluster_id | aws_rds_cluster |
+| this_rds_cluster_instance_endpoints | aws_rds_cluster_instance |
+| this_rds_cluster_master_password | The master password |
+| this_rds_cluster_master_username | The master username |
+| this_rds_cluster_port | The port |
+| this_rds_cluster_reader_endpoint | The cluster reader endpoint |
+| this_security_group_id | aws_security_group |
```

Comment:
What's the deal with prepending every output with `this_`? In English this sounds kinda silly. Like there is no `that` cluster to justify the specification, and even if there was, to use the words `this` is not very....technical?

Reply:
`this` is the default name of the resource.

All outputs are following the naming structure as _RESOURCE_NAME_+_TYPE_+_ATTRIBUTE_, so that such names are always unique in the whole module and descriptive enough for human.

Reply:
OK! Sounds and look daft to me but I'll take your word for it 😎